wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/templates/settings-edit.php

```diff
@@ -96,13 +116,28 @@
 	<tr>
 		<th><label for="default_sort[deepl_api_key]">
 				<?php esc_html_e( 'Deepl Free API Key', 'glotpress' ); ?>
+			</label>
+					<?php
+					if ( gp_array_get( $gp_external_translations, 'deepl_translations_used', 0 ) > 0 ) {
+						echo '<br>';
+						echo '<small>';
+						/* translators: %d: number of OpenAI translations used. */
+						echo esc_html( sprintf( _n( '%d Deepl translation used', '%d Deepl translations used', 'glotpress' ), gp_array_get( $gp_external_translations, 'deepl_translations_used', 0 ) ) );
```

Comment:
They themselves always spell DeepL with a capital L at the end. Could you replace all occurences of outputted text so that it conforms to that?

Reply:
Also, could we, to make it more clear add a "with modifications" here?